README.md

```diff
@@ -1,30 +1,130 @@
+    cdef long long i_0
+    cdef long long i_1
+    cdef long i_2
+    data = {}
+    data['order_id'] = read.read_string(fo)
+    data['trades'] = []
+
+    i_1 = read.read_long(fo)
+    while i_1 != 0:
+        if i_1 < 0:
+            i_1 = -i_1
+            read.read_long(fo)
+        for i_0 in range(i_1):
+            i_2 = read.read_int(fo)
+            if i_2 == 0:
+                val_2 = read.read_string(fo)
+            if i_2 == 1:
+                val_2 = read.read_int(fo)
+            data['trades'].append(val_2)
+        i_1 = read.read_long(fo)
+    return data
+```
+
+
 **Usage Example:**
-1. Put all your schemata into a schemata root folder following this pattern.<br>
-`[schemata_root_folder]/[namespace]/[schema_name]/[schema_version]/schema.yaml`<br>
-It is essential that you follow this pattern and safe your schemata in `schema.yaml` files.
-
-2. Run `cerializer_handler.update_cerializer` with a list of all your schemata root directories.<br>
-eg. `cerializer_handler.update_cerializer(['cerializer/tests/schemata'])`<br>
-This will generate an importable module for each schema that contains two methods: `serialize` and `deserialize`.<br>
-This module can be then imported as:<br>
- `import [schema_name]_[schema_version]`<br>
- and then used accordingly<br>
-3. Use the module accordingly.<br>
-  eg.<pre><code>
- import student_schema_1
- import io<br>
- output = io.BytesIO()
- data_recod = {
+1. Create an instance of CerializerSchemata
+For initializing CerializerSchemata it is necessary to supply a list of tuples in form of (schema_identifier, schema)
+schema touple = (namespace.schema_name, {schema})
+ eg.
+ ```python
+import cerializer.schema_handler
+import os
+import yaml
+
+def list_schemata():
+    # iterates through all your schemata and yields schema_identifier and path to schema folder
+    raise NotImplemented
+
+def schemata() -> cerializer.schema_handler.CerializerSchemata:
+    schemata = []
+	for schema_identifier, schema_root in list_schemata():
+		schema_tuple = schema_identifier, yaml.unsafe_load( # type: ignore
+			open(os.path.join(schema_root, 'schema.yaml'))
+		)
+		schemata.append(schema_tuple)
+	return cerializer.schema_handler.CerializerSchemata(schemata)
+```
+
+2. Create an instance of Cerializer for each of your schemata by calling `cerializer_handler.Cerializer` .
+eg. `cerializer_instance = cerializer_handler.Cerializer(cerializer_schemata, schema_namespace, schema_name)`
+This will create an instance of Cerializer that can serialize and deserialize data in the particular schema format.
+
+3. Use the instance accordingly.
+  eg. 
+  ```python
+ data_record = {
     'name': 'Matej',
     'age': 22
  }
- student_schema_1.serializer(data_record, output)
- print(output.getvalue())
- </code></pre>
+ cerializer_instance = cerializer.cerializer_handler.Cerializer(cerializer_schemata, 'school', 'student')
+ serialized_data = cerializer_instance.serialize(data_record)
+ print(serialized_data.getvalue())
+```
```

Comment:
Maybe you could add a section with the results of the benchmark here. To show that this is worth it. What do you think?